pygfunction/pipes.py

```diff
@@ -370,7 +368,7 @@ def coefficients_temperature(self, z, m_flow, cp, nSegments):
 
         """
         # method_id for coefficients_temperature is 5
-        method_id = 5
+        # method_id = 5  Unused: check
```

Comment:
```method_id``` is not used here. Check usage.

Reply:
This one should probably stay, as there may be a need for memory in the future.